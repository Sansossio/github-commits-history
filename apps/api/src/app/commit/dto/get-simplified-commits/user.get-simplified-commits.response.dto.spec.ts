import { UserGetSimplifiedCommitsResponseDTO } from './user.get-simplified-commits.response.dto'

describe('UserGetSimplifiedCommitsResponseDTO', () => {
  it('should be defined', () => {
    expect(UserGetSimplifiedCommitsResponseDTO).toBeDefined()
  })

  it('should match snapshot', () => {
    const user: any = {
      id: 123,
      login: 'username',
      avatar_url: 'https://google.com',
      url: 'https://yahoo.com'
    }

    expect(UserGetSimplifiedCommitsResponseDTO.fromGithubResponse(user)).toMatchSnapshot()
  })
})