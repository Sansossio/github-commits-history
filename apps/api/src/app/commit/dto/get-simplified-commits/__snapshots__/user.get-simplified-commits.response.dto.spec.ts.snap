// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserGetSimplifiedCommitsResponseDTO should match snapshot 1`] = `
UserGetSimplifiedCommitsResponseDTO {
  "avatar_url": "https://google.com",
  "id": 123,
  "login": "username",
  "url": "https://yahoo.com",
}
`;
